README.md

```diff
@@ -14,13 +14,12 @@ There will never be a guide that will give you perfect results. Each server has
 # Preparations
 
 ## Server jar
-Your choice of server software can make a huge difference in performance and api possibilities. There are currently multiple viable popular server jars, but there are also a few that you should stay away from for various reasons.
+Your choice of server software can make a huge difference in performance and API possibilities. There are currently multiple viable popular server JARs, but there are also a few that you should stay away from for various reasons.
 
 Recommended top picks:
 * [Paper](https://github.com/PaperMC/Paper) - The most popular server software that aims to improve performance while fixing gameplay and mechanics inconsistencies.
 * [Tuinity](https://github.com/Spottedleaf/Tuinity) - Paper fork that includes even more high-performance patches.
 * [Purpur](https://github.com/pl3xgaming/Purpur) - Tuinity fork focused on features and the freedom of customization.
-* [Airplane](https://github.com/Technove/Airplane) - Tuinity fork that aims to further improve server performance.
```

Comment:
I do not agree with removing notable forks from this list.
The point of this guide is to educate and inform people of available options, removing airplane from here would hurt guide's objectivity. Unfortunately you disallowed edits so I cannot fix this myself.

Reply:
@YouHaveTrouble sorry for the inconvenience of disallowing edits. If removing Airplane is not an option, would adding a note that DEAR can sometimes cause problems be viable?

Reply:
I will add such note later on

Reply:
@YouHaveTrouble re-added, with a note, hopefully the note isn't too negative. feel free to suggest changes